source/upgrade.txt

```diff
@@ -0,0 +1,123 @@
+.. _php-upgrade:
+
+=======================
+Upgrade Driver Versions
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatibility, backwards compatibility
+
+Overview
+--------
+
+On this page, you can learn how to upgrade your driver to a new version. This
+page also includes the changes you must make to your application when you 
+upgrade to a new version of the {+php-library+}.
+
+How to Upgrade
+--------------
+
+Before you upgrade, perform the following actions:
+
+- Ensure the new {+driver-short+} version is compatible with the {+mdb-server+} versions
+  your application connects to and the PHP version your
+  application compiles with. For version compatibility information, see the
+  :ref:`{+php-library+} Compatibility <php-compatibility>`
+  page.
+- Address any breaking changes between the driver version
+  your application is using and your planned upgrade version in the
+  :ref:`Breaking Changes <php-breaking-changes>` section.
+
+.. tip::
+
+   To ensure compatibility across {+mdb-server+} versions when
+   upgrading driver versions, use the :ref:`{+stable-api+} <php-stable-api>`.
+
+To upgrade your driver version, replace ``<version-number>`` with the version number 
+you want to upgrade to and run the following command in your application's 
+directory:
+
+.. code-block:: bash
+
+   composer require mongodb/mongodb:<version-number>
+
+Installation instructions for the ``mongodb`` extension may be found in the 
+:php:`PHP.net documentation <manual/en/mongodb.installation.php>`.
```

Comment:
We should actually list the instructions on how to install the extension as well. This can be done by running the following:
```
pecl upgrade mongodb-<version-number>
```

The note about the PHP documentation can be amended to read "Detailed installation instructions [...]"

We should highlight that while the major and minor version numbers of the extension and library are in sync, the patch version is not.

Reply:
Added explanation about upgrading for different versions. Let me know what you think. thanks!